test/train-sets/ref/squarecb_save.stderr

```diff
@@ -12,17 +12,17 @@ Input label = cb
 Output pred = action_probs
 average  since         example        example  current  current  current
 loss     last          counter         weight    label  predict features
-0.000000 0.000000            1            1.0 4:0:0.14        0:0...       42
-0.000000 0.000000            2            2.0 0:0:0.14        0:0...       42
-0.000000 0.000000            4            4.0 1:0:0.14        0:0...       42
-0.000000 0.000000            8            8.0 3:0:0.14        0:0...       42
-0.000000 0.000000           16           16.0 3:0:0.14        0:0...       42
-0.000000 0.000000           32           32.0 3:0:0.14        0:0...       42
-0.127219 0.254438           64           64.0 0:0:0.77        0:-0.259838...       42
+0.000000 0.000000            1            1.0 4:0:0.14        0:0.142857...       42
+0.000000 0.000000            2            2.0 0:0:0.14        0:0.142857...       42
+0.000000 0.000000            4            4.0 1:0:0.14        0:0.142857...       42
+0.000000 0.000000            8            8.0 3:0:0.14        0:0.142857...       42
+0.000000 0.000000           16           16.0 3:0:0.14        0:0.142857...       42
+0.000000 0.000000           32           32.0 3:0:0.14        0:0.142857...       42
+-0.250000 -0.500000           64           64.0 0:0:0.77        0:0.765981...       42
```

Comment:
can we add predictions file to this test/reference?

Reply:
Yes added